include/dom/components/datasources/offsettime.hpp

```diff
@@ -0,0 +1,88 @@
+#ifndef DOT_MESSAGE_OFFSETTIME_HPP
+#define DOT_MESSAGE_OFFSETTIME_HPP
+
+#include <string>
+
+constexpr int kMaxHourDay = 23;
```

Comment:
Revisar que no tingui efectes col·lateras no comentats
```suggestion
constexpr int kMaxHourDay = 24;
```

Reply:
corregit